src/test/java/seedu/application/logic/commands/CommandResultTest.java

```diff
@@ -13,26 +11,26 @@ public void equals() {
         CommandResult commandResult = new CommandResult("feedback");
 
         // same values -> returns true
-        assertTrue(commandResult.equals(new CommandResult("feedback")));
-        assertTrue(commandResult.equals(new CommandResult("feedback", false, false)));
+        assertEquals(commandResult, new CommandResult("feedback"));
```

Comment:
More accurate use of assert methods rather than trying to force feed test results. Very sharp on identifying that after u implemented the test cases